internal/cmd/measurement_schema/client.go

```diff
@@ -0,0 +1,284 @@
+package measurement_schema
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/influxdata/influx-cli/v2/internal"
+	"github.com/influxdata/influx-cli/v2/internal/api"
+)
+
+type Client struct {
+	BucketApi        api.BucketsApi
+	BucketSchemasApi api.BucketSchemasApi
+	CLI              *internal.CLI
+}
+
+type orgBucketID struct {
+	OrgID    string
+	BucketID string
+}
+
+func (c Client) resolveMeasurement(ctx context.Context, ids orgBucketID, name string) (string, error) {
```

Comment:
Not sure if it matters, but should these methods take pointer receivers?

Reply:
I've kept it as a value receiver to indicate that `Client` is immutable. `Client` ultimately serves as a namespace for the functions.